components/config/definition.rst

```diff
@@ -395,6 +395,45 @@ with ``append()``::
 This is also useful to help you avoid repeating yourself if you have sections
 of the config that are repeated in different places.
 
+The example results in the following:
+
+.. code-block:: yaml
+
+    database:
+        connection:
+            driver:               ~ # Required
+            host:                 localhost
+            username:             ~
+            password:             ~
+            memory:               false
+            parameters:           # Required
+    
+                # Prototype
+                name:
+                    value:                ~ # Required
+
+.. code-block:: xml
+
+    <config>
```

Comment:
yes should, but i executed this code from the doc and this was the result.

![bildschirmfoto 2015-08-10 um 18 24 29](https://cloud.githubusercontent.com/assets/995707/9176685/161c92f0-3f8d-11e5-84ca-74ff2cc253db.png)